### ISSUE TITLE
Is there any plans to commit FLUX training to the main branch?

### ISSUE THREAD

**Issue by modestasgrz** [bmaltais/kohya_ss#2922]
Hi,

I was wondering whether you have plans to commit flux training implementation to the main branch?

I wanted to write my own simplified flux training implementation GUI, but I ran into difficulties integrating flux training scripts. In my case, when arguments are being passed through subprocess command line, I get an unrecognized arguments error. When I try to launch training directly by incorporating an additional method in flux training script, that collects argument values from the training config and launches training, I get "NotImplementedError: Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta to a different device" error, which, I assume, origins from insufficient VRAM, which is nonsense, as I am using RTX4090 and passing params for 12GB VRAM training (optimizer - Adafactor, split mode etc.)

I suppose, the reason that flux training scripts are not yet merged with the main branch is because of the currently existing issues (?).
Last time I integrated SDXL training (from the main branch) into my GUI, which ran without such errors.

Can you tell me whether my assumptions are correct? Or am I missing something?

**Comment by bmaltais** [comment]
Yes, as soon as Kohya merge it to his main branch I will... but it might take a little while as he will merge it 1st into his dev branch... and I will do the same for the GUI...
